### DISCUSSION TITLE
Notify me when someone is assigned to my Issue

### DISCUSSION THREAD

**Discussion by ghost** [community/community#29578] [Projects and Issues]
I'd like a setting to notify me when someone is assigned to my Issue.

**Comment by mazkdevf** [comment]
Well what is you issue?

**Reply by ghost** [reply]
This is a feature request. As I understand it, the "Issue" category is for anything relevant to Issues as a feature, not any issues with any GitHub product.

> We encourage you to [open a discussion](https://github.com/orgs/community/discussions) if you have suggestions for how we can improve our products.

https://github.com/community/community
